### PR TITLE
[ch19928] Implement deploy command with a review env chart

### PR DESCRIPTION
**Pull request by victorpoluceno**
Add a deploy command that can be used to install the review env chart. 

The helm chart was originally developed internally. In this PR I'm mostly copying and changing the name.

### CODE REVIEW

File: .codefresh/build/spec.yml [hunk 1]
```diff
@@ -34,5 +34,5 @@ spec:
     location: git
     repo: FindHotel/cf-review-env
     path: ./.codefresh/build/pipeline.yml
-    revision: master
+    revision: ch19928
```
Comment (victorpoluceno) [inline .codefresh/build/spec.yml:37]: I will move this back to master before merging.

File: deploy/deploy [hunk 2]
```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+export NAMESPACE=re-$NAME
+kubectl config use-context $KUBE_CONTEXT
+kubectl create namespace $NAMESPACE --dry-run -o yaml | kubectl apply -f -
+kubectl create secret generic $NAMESPACE --from-env-file=/codefresh/volume/secrets.env --dry-run -o yaml --save-config=true --namespace $NAMESPACE | kubectl apply -f -
```
Comment (rodrigo-findhotel) [inline deploy/deploy:5]: This parameter will be deprecated `--dry-run` and the new is  `--dry-run=client`

Comment (victorpoluceno) [reply deploy/deploy:5]: The kubectl version that we have installed is bit old and doesn't support this param. I will leave like that for now.

Comment (rodrigo-findhotel) [reply deploy/deploy:5]: But can we update the client version?
Do we have control of this version?

**Pull request merged**